crypto/src/pubkey_bytes.rs

```diff
@@ -67,18 +67,31 @@ impl<T, const N: usize> Default for PublicKeyBytes<T, N> {
     }
 }
 
+impl<T: VerifyingKey, const N: usize> FromStr for PublicKeyBytes<T, N> {
+    type Err = anyhow::Error;
+
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        let s = s.strip_prefix("0x").unwrap_or(s);
+        let value = hex::decode(s)?;
+        Self::from_bytes(&value[..]).map_err(|_| anyhow::anyhow!("byte deserialization failed"))
+    }
+}
+
+impl<T: VerifyingKey, const N: usize> Copy for PublicKeyBytes<T, N> {}
```

Comment:
is the rationale behind this that we want to over the Copy capabilities only for the structs that extend the `VerifyingKey`? If yes, what's the reason ?

Reply:
Sorry for responding to this late, but a problem was that type T could possibly not have the Copy trait, which will make the compiler throw an error.